### ISSUE TITLE
Handlebars comments on same line as HTML tag breaks linter

### ISSUE THREAD

**Issue by pgengler** [rwjblue/ember-cli-template-lint#195]
`ember-cli-template-lint` version 0.5.2

Using a template like this:
```
<div> {{! I'm a comment }}
</div>
```

Causes linting to fail with this error:
```
lint-error/templates/application.hbs should pass TemplateLint.
undefined: Cannot read property 'original' of undefined (lint-error/templates/application): 
`TypeError: Cannot read property 'original' of undefined
    at BasePlugin.BlockIndentation.validateBlockChildren (/Users/phil.gengler/tmp/lint-error/node_modules/ember-cli-template-lint/node_modules/ember-template-lint/lib/rules/lint-block-indentation.js:246:31)
    at BasePlugin.BlockIndentation.process (/Users/phil.gengler/tmp/lint-error/node_modules/ember-cli-template-lint/node_modules/ember-template-lint/lib/rules/lint-block-indentation.js:118:10)
    at BasePlugin.ElementNode (/Users/phil.gengler/tmp/lint-error/node_modules/ember-cli-template-lint/node_modules/ember-template-lint/lib/rules/lint-block-indentation.js:109:14)
    at /Users/phil.gengler/tmp/lint-error/node_modules/ember-cli-template-lint/node_modules/ember-template-lint/lib/rules/base.js:290:21
    at visitNode (/Users/phil.gengler/tmp/lint-error/node_modules/ember-cli-template-lint/node_modules/ember-template-lint/node_modules/glimmer-engine/dist/node_modules/glimmer-syntax/lib/traversal/traverse.js:8:32)
    at visitArray (/Users/phil.gengler/tmp/lint-error/node_modules/ember-cli-template-lint/node_modules/ember-template-lint/node_modules/glimmer-engine/dist/node_modules/glimmer-syntax/lib/traversal/traverse.js:63:22)
    at visitKey (/Users/phil.gengler/tmp/lint-error/node_modules/ember-cli-template-lint/node_modules/ember-template-lint/node_modules/glimmer-engine/dist/node_modules/glimmer-syntax/lib/traversal/traverse.js:46:9)
    at visitNode (/Users/phil.gengler/tmp/lint-error/node_modules/ember-cli-template-lint/node_modules/ember-template-lint/node_modules/glimmer-engine/dist/node_modules/glimmer-syntax/lib/traversal/traverse.js:24:13)
    at Object.traverse (/Users/phil.gengler/tmp/lint-error/node_modules/ember-cli-template-lint/node_modules/ember-template-lint/node_modules/glimmer-engine/dist/node_modules/glimmer-syntax/lib/traversal/traverse.js:105:5)
    at BasePlugin.transform (/Users/phil.gengler/tmp/lint-error/node_modules/ember-cli-template-lint/node_modules/ember-template-lint/lib/rules/base.js:76:17)`
```

Using the other comment style (`{{!-- I'm a comment --}}`) causes the same error. Moving the comment to a separate line clears things up.

**Comment by kellyselden** [comment]
This is resolved in ember-template-lint v0.6.4

**Comment by pgengler** [comment]
Confirmed! Thanks!
